src/inhibitors/blacklist.js

```diff
@@ -3,6 +3,7 @@ const { Inhibitor } = require('klasa');
 module.exports = class extends Inhibitor {
 
 	async run(msg) {
+		if (msg.author === this.client.owner) return;
```

Comment:
For performance, I'd suggest `this.client.options.ownerID`, as `this.client.owner` is a getter:

https://github.com/dirigeants/klasa/blob/bb46f404a40448c880f0c0d4662a894f27dc7977/src/lib/Client.js#L296-L298